packages/web3-utils/types/tests/sha3-test.ts

```diff
@@ -17,6 +17,7 @@
 /**
  * @file sha3-tests.ts
  * @author Josh Stevens <joshstevens19@hotmail.co.uk>
+ * @author Prince Sinha <sinhaprince013@gmail.com>
```

Comment:
This whole change file can be reverted now no point committing just this, changing the git history for this file when you have not touched it.

Reply:
Ya sorry, I forgot to remove that. Extremely sorry. @joshstevens19

Reply:
No worries easy thing to forget 👍🤘🏿